lib/cli/index.js

```diff
@@ -66,7 +66,7 @@ function cli(options) {
   var ui = new UI({
     inputStream:  options.inputStream,
     outputStream: options.outputStream,
-    ci:           process.env.CI,
+    ci:           process.env.CI || /^(dumb|emacs)$/.test(process.env.TERM),
```

Comment:
Should we use `process.stdout.isTTY` instead of `process.env.TERM`?

https://nodejs.org/api/process.html#process_process_stdout
https://nodejs.org/api/tty.html#tty_tty